pypeit/tests/test_flatfield.py

```diff
@@ -60,8 +60,12 @@ def test_run():
     par = pypeitpar.FrameGroupPar(frametype)
     flatField = flatfield.FlatField(spectrograph, par, det=1, tilts_dict=tilts_dict,
                                     tslits_dict=tslits_dict.copy())
+
+    # TODO mstrace is no longer stored in the master file so this needs to be run from an input file which must
```

Comment:
I'm not spending time on tests right now. If I do that, I'll never get anything done because this adds massive overheads to every PR for me given the significance of my changes. We can clean up the tests later.